docs/reference/ml/trained-models/apis/put-trained-model-vocabulary.asciidoc

```diff
@@ -67,7 +67,7 @@ PUT _ml/trained_models/elastic__distilbert-base-uncased-finetuned-conll03-englis
   ]
 }
 --------------------------------------------------
-// NOTCONSOLE
+// TEST[s/\.\.\./"[PAD]"/ skip:TBD]
```

Comment:
I don't think it's realistic to ever run that test, as it would require that the `elastic__distilbert-base-uncased-finetuned-conll03-english` has been imported into the docs test cluster, and that is an impractically heavy operation for the docs tests. So I think it's best to just leave this test skipped.